_imagingft.c

```diff
@@ -75,6 +78,32 @@ typedef struct {
 
 static PyTypeObject Font_Type;
 
+typedef struct {
+	void* raqm;
```

Comment:
Tabs to spaces

Reply:
yeah, I'm on a new vm and the emacs isn't setup right yet.